src/redis.go

```diff
@@ -60,6 +61,8 @@ func main() {
 		os.Exit(0)
 	}
 
+	args.Hostname = hostnameIpv6Formater(args.Hostname)
+
 	conn, err := newRedisCon(args.Hostname, args.Port, args.UnixSocketPath, args.Password)
```

Comment:
The change above would require to change this function to receive host and port together, which should be fine by me.